recipes/labours/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "labours" %}
+{% set version = "10.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/src-d/hercules/archive/v{{ version }}.tar.gz
+  sha256: d152b34843452f62e9e4d4e287552b5505c2f4fd7289356f6aed0afe209e08b3
+  patches:
+    - 0001_win_setup_py_handle_os_error.patch  # [win]
+
+build:
+  number: 0
+  entry_points:
+    - labours=labours.__main__:main
+  noarch: python
+  script: "cd python && {{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - clint >=0.5.1,<1.0
+    - fastdtw >=0.3.2,<2.0
+    - hdbscan >=0.8.0,<2.0
+    - lifelines >=0.20.0,<2.0
+    - matplotlib >=2.0,<4.0
+    - munch >=2.0,<3.0
+    - numpy >=1.12.0,<2.0
+    - pandas >=0.20.0,<1.0
+    - protobuf >=3.5.0,<4.0
+    - python >=3.6
+    - python-dateutil >=2.6.0,<3.0
+    - pyyaml >=3.0,<5.0
+    - scipy >=0.19.0,<1.3.0
+
+test:
+  files:
+    # generated via: hercules --burndown --burndown-people --devs --pb \
+    #   https://github.com/conda-forge/conda-smithy > test.pb
+    - test.pb.bz2
+  requires:
+    - bzip2
+  imports:
+    - labours
+  commands:
+    - labours --help
+    - bunzip2 test.pb.bz2
+    - labours -i test.pb -o test-bpr -m burndown-project
+    - labours -i test.pb -o test-def -m devs-efforts
+    - labours -i test.pb -o test-cma -m churn-matrix
+    - labours -i test.pb -o test-ovn -m old-vs-new
+    - labours -i test.pb -o test-own -m ownership
+
+about:
+  home: "https://github.com/src-d/hercules"
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE.md
```

Comment:
Could you please raise an issue on this project to also package the license?

Reply:
Opened https://github.com/src-d/hercules/pull/288

Reply:
removed the recipe